ui/pages/onboarding-flow/metametrics/metametrics.js

```diff
@@ -1,6 +1,8 @@
 import React, { useContext } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { useHistory } from 'react-router-dom';
+// eslint-disable-next-line import/no-restricted-paths
+import { getPlatform } from '../../../../app/scripts/lib/util';
```

Comment:
`getPlatform` should definitely be under `shared/`, that would address this lint error.

Good call on ignoring it for now, we can fix this separately.